scala3doc/src/dotty/dokka/model/api/membersUtils.scala

```diff
@@ -0,0 +1,27 @@
+package dotty.dokka.model.api
+
+import org.jetbrains.dokka.model.DModule
+import scala.jdk.CollectionConverters.{ListHasAsScala, SeqHasAsJava}
+import dotty.dokka.model.api._
+
+extension (m: DModule)
+    def visitMembers(callback: Member => Unit): Unit = 
+        def visitClasslike(c: Member, callback: Member => Unit): Unit =
```

Comment:
you do not need to duplicate `callback` parameter `visitClasslike` has access to top level one